config/install/views.view.iiif_manifest.yml

```diff
@@ -14,6 +14,8 @@ dependencies:
     - serialization
     - taxonomy
     - user
+_core:
```

Comment:
My understanding is we're leaving the hashes out of the configs. Is that correct?

Reply:
Removed.